apps/acus/pages/report-admin.tsx

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react'
 import type { NextPage } from 'next'
 import { Reports, ReportRecord } from 'amber/views/Reports'
+import { Perms } from 'amber/components/Auth'
 
 const reports: ReportRecord[] = [
   { name: 'Membership' },
   { name: 'Game' },
   { name: 'GM' },
   { name: 'Game And Players' },
   { name: 'Games Scheduler' },
+  { name: 'Game Choices', perm: Perms.GameAdmin },
```

Comment:
Did you mean this to be `Perms.PlayerAdmin` since that's the new perm you've added elsewhere.

Reply:
Yeah

Reply:
Fixed